### DISCUSSION TITLE
Why did Antlr have a separate com.springsource.antlr package on maven central for versions 2.7.6 and 2.7.7?

### DISCUSSION THREAD

**Discussion by ikannak** [antlr/antlr4#4448] [Q&A]
I noticed that in the dependency management of a project I'm preparing a library upgrade for, we are using Hibernate 5.4.12, which has a dependency on Antlr 2.7.7. That said, I also noticed that we're using the Maven Central com.springsource.antlr package of this version, and not the basic antlr package. My coworkers can't remember why we originally did this, so I thought I'd ask the community why this specific package was created in Maven Central. I want to be sure that there wasn't a special reason why we were using this specific package and not the basic package. My gut feeling is that we needed a version that's compatible with OSGi since we use Karaf, but I wantd to do a sanity check first.

**Comment by ericvergnaud** [comment]
I believe that antlr4 is now OSGI compliant.
